avogadro/rendering/geometryvisitor.h

```diff
@@ -74,6 +111,11 @@ class GeometryVisitor : public Visitor
 
   std::vector<Vector3f> m_centers;
   std::vector<float> m_radii;
+#ifdef _3DCONNEXION
```

Comment:
Why are you saving all of these? This seems like a really bad idea when someone has 10^5 or 10^6 atoms and bonds.

Reply:
I agree, the hit testing part introduced by me is redundant and it can be memory consuming as you noted. I have reworked it today.